[D 20221213 00:37:35 __init__:281] matplotlib data path: /home/user/anaconda3/envs/py38/lib/python3.8/site-packages/matplotlib/mpl-data
[D 20221213 00:37:35 __init__:281] CONFIGDIR=/home/user/.config/matplotlib
[D 20221213 00:37:35 __init__:1456] interactive is False
[D 20221213 00:37:35 __init__:1457] platform is linux
[D 20221213 00:37:35 __init__:281] CACHEDIR=/home/user/.cache/matplotlib
[D 20221213 00:37:35 font_manager:1629] Using fontManager instance from /home/user/.cache/matplotlib/fontlist-v330.json
